src/proxy-page/steps/addSlides.ts

```diff
@@ -29,7 +29,7 @@ export default (): Step => (context: ContextService): void => {
       sections.forEach((section: cheerio.CheerioAPI) => {
         const className = section('body').children().first().attr('class') ?? '';
         // Based on the 'tag' name of the first element we will design the slide format
-        switch (section('body').first().children().get(0).tagName) {
+        switch (section('body').first().children().get(0)?.tagName) {
```

Comment:
@krystiankedra 
I have added this check to avoid a bug that happened when several horizontal lines where together without any content in between.